scripts/add-worker-arch.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+import os
+import platform
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+
+
+def main():
+    from kobo.hub.models import Arch, Worker  # noqa:  E402
```

Comment:
I'll apply this patch during next rebase/merge.
```suggestion
    from kobo.hub.models import Arch, Worker  # noqa: E402
```

Reply:
Addressed in 7b998de.